src/examples/Duplicate.cpp

```diff
@@ -123,7 +123,7 @@ void tvgDrawCmds(tvg::Canvas* canvas)
         file.close();
 
         auto picture1 = tvg::Picture::gen();
-        if (picture1->load(data, 200, 300, true) != tvg::Result::Success) return;
+        if (picture1->load(data, 200, 300, false, true) != tvg::Result::Success) return;
```

Comment:
Come on, could you please fix all other usages as well?

Reply:
Yes.. I know... On the way